data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -24,37 +24,37 @@
             story= _ "The dwarves were more than ready for them, however, and as soon as the humans got through the defenses the well-prepared dwarves fell on the battle-weary and bloodied orcs."
         [/part]
         [part]
-            story= _ "Wave after wave of orcs crashed onto the dwarven defenses and shattered."
+            story= _ "Wave after wave of orcs crashed onto the dwarven defenses only to be shattered."
         [/part]
         [part]
             story= _ "As the battle raged on, Tallin again made council with Hamel."
         [/part]
 
         {STORY_PART_HAMEL (_ "How did ye fare?")}
 
-        {STORY_PART_TALLIN (_ "It was fierce and bloody. We fought them for three days and nights with no end in sight. Every time we killed an orc, it seemed that two more would take his place. We most assuredly cannot win fighting like this.")}
+        {STORY_PART_TALLIN (_ "It was fierce and bloody. We fought them for three days and nights. Every time we killed an orc, it seemed that two more would take his place. We most assuredly cannot win fighting like this.")}
 
         {STORY_PART_HAMEL (_ "So you withdrew. Rightly done, me lad. Our prepared defenses will be able to hold the orcs off far longer and with less loss than ye could ha’ done in the open field. I see that my training of you hasna’ been for naught.")}
 
         {STORY_PART_TALLIN (_ "And I am grateful — but what should we do from here? The number of humans and dwarves is limited, unlike the orcs whose numbers seem unlimited.")}
 
-        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ha’ anything other than a brief respite.")}
+        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ever ha' peace.")}
 
         {STORY_PART_HAMEL (_ "Orcs are fickle; if we negotiated a treaty with them — assuming they would accept — it would just be broken as soon as the next warlord arose among them.")}
 
         {STORY_PART_TALLIN (_ "Then what shall we do? Can we assassinate this Rakshas fellow and throw them into disarray once more?")}
 
         #po: "coil" meant, in Middle and Early Modern English, a difficulty
         #po: or problem.
-        {STORY_PART_HAMEL (_ "That will be very difficult. We did that last time and they will no doubt be alert to that threat now. Anyway, it would be but a temporary solution. Soon another strong leader would emerge and we’d find oursel’ back in the same coil.")}
+        {STORY_PART_HAMEL (_ "That will be very difficult. We did that last time, and they will no doubt be alert to that threat now. Anyway, it would be but a temporary solution. Soon another strong leader would come along, and we’d find oursel’ back in the same coil.")}
```

Comment:
The first comma isn't needed.

Reply:
Ok.

Reply:
Please remove the first comma you forgot lol.